cmd/agent/app/builder.go

```diff
@@ -71,12 +68,7 @@ type Builder struct {
 	HTTPServer HTTPServerConfiguration  `yaml:"httpServer"`
 	Metrics    jmetrics.Builder         `yaml:"metrics"`
 
-	// These 3 fields are copied from tchreporter.Builder because yaml does not parse embedded structs
-	CollectorHostPorts   []string `yaml:"collectorHostPorts"`
-	DiscoveryMinPeers    int      `yaml:"minPeers"`
-	CollectorServiceName string   `yaml:"collectorServiceName"`
-
-	tchreporter.Builder
+	tchreporter.Builder `yaml:",inline"`
```

Comment:
this is awesome. Could you search the rest of the code for this though? I think there may be other examples of this copy-pasta.

btw, why comma before `inline`?

Reply:
>btw, why comma before inline?

It declares an empty yaml tag for `tchreporter.Builder`. Simply using `inline` doesn't work.

 >Could you search the rest of the code for this though? I think there may be other examples of this copy-pasta.

I wasn't able to come up with a good way of searching for anonymous structs; Nothing else seems to have similar comments.